cosmwasm/devnet/config/config.toml

```diff
@@ -12,7 +12,7 @@
 
 # TCP or UNIX socket address of the ABCI application,
 # or the name of an ABCI application compiled in with the Tendermint binary
-proxy_app = "tcp://127.0.0.1:26656"
+proxy_app = "tcp://127.0.0.1:26658"
```

Comment:
Just out of curiosity, every other port was incremented by 1 and this is by 2?  Are there 2 ports being used by the proxy?

Reply:
maybe because it's terra 2? heh. this is another straight copy of https://github.com/terra-money/LocalTerra/tree/main/config